website/static/js/pages/home-page.js

```diff
@@ -60,6 +68,15 @@ $(document).ready(function(){
 
                     ]
                 )),
+                /*
```

Comment:
Yep, that's intentional. Whenever the banner gets moved below the dashboard again this block will just need to be uncommented and everything will be as it was. It's also why I didn't delete/change preregPlugin.js and instead made preregBannerPlugin.js.